point_pattern.py

```diff
@@ -109,7 +115,12 @@ def check_coincident(a, b):
     equal : bool
             Whether the points are equal
     """
-    return None  # Add the logic to check if coincident here
+    #return None # Add the logic to check if coincident here
+
+    if a[0]==b[0] and a[1]==b[1]:
+        coincident = True
+        return coincident
+
```

Comment:
Thank you! 


Sent from my Verizon Wireless 4G LTE smartphone

-------- Original message --------
From: jlaura <notifications@github.com> 
Date: 02/11/2016  7:49 AM  (GMT-07:00) 
To: Geospatial-Python/assignment_03 <assignment_03@noreply.github.com> 
Cc: HerczegAdam <herczegadam@yahoo.com> 
Subject: Re: [assignment_03] Almost (#12) 

In point_pattern.py:
> @@ -109,7 +115,12 @@ def check_coincident(a, b):
>      equal : bool
>              Whether the points are equal
>      """
> -    return None  # Add the logic to check if coincident here
> +    #return None # Add the logic to check if coincident here
> +
> +    if a[0]==b[0] and a[1]==b[1]:
> +        coincident = True
> +        return coincident
> +

Super close here.  Nice job!  Right now, if coincident is True, this returns True (exactly as it should).  What about if coincident is False?  Right now, this does not return anything, e.g. None.  We will talk a bit more about if statments later, so here is a way to do it without an if:

return a[0]==b[0] and a[1]==b[1]

Alternatively, if you want to use an if statement, you could do:

coincident = False
if a[0]==b[0] and a[1]==b[1]:
    coincident = True

return coincident

—
Reply to this email directly or view it on GitHub.